### DISCUSSION TITLE
Discord link

### DISCUSSION THREAD

**Discussion by clovewilliam** [carperbr/vocal-remover-frame-transformer#5] [Q&A]
I'm late to the game and the discord link says it is expired. If the discord server is still active, would it be possible to get a new invite link for it?

Thanks!

**Comment by carperbr** [comment]
Thanks for letting me know, just updated it! https://discord.gg/TftEyYJG

**Reply by clovewilliam** [reply]
Thank you!

**Comment by MSHClivingston** [comment]
Hey, Benjamin. I saw that your Youtube got nuked. Do you still make those metal instrumentals and, um... share them somewhere?

Apologies if this is not the right place to ask, but I don't know how else to contact you.
